packages/wallets/spot/src/adapter.ts

```diff
@@ -0,0 +1,246 @@
LqUjDINKBRjEA7IKBlU5EHcA4cGZlO5GEH/QKfRg2q2wr3jjYmu8xseh59gk4aAyHq0YaZtJ0XkmW39EJm/WuMaM6bemNXdPZEhwSPxCjdWxFW7HmG5wgQjmGhMqVx9pbB4lo/KyOP8nh1+aRXnRy53faOxY/8o3uK3AMAgp7Tf7N+pfocT+128EXvy4cMeNgGP2SCN5WxM87EAr5mMbmXftuIFcE2RI/gSf0MsHAnrowr7GQ/elvFDzp9WEkM/NBUYCJU5QRrn16NTv4UiPb9CM4oP2836/vEV+TUbtW9zA8G9ykmz/Lz/s7rjEz3prfbmweLDtdef98PL27VZ+5v9oFPRmPWJzCqe2nnM3nv2Vl3mbgHzOKFjuWZ+ciIh+tq9gNk8Ac99Nn3udKeAwgb1iVf3e8dEA6WkJnuBpYsA48AYaDYc+7XKLuYL5MkOIQTkYoLS4UPvLhGHmZKL/b10LGi5acVQkORcNlBU8nGot2oq4xMNexy1i2QL9jy2WHkla5y0mLxDxdv/8FboxKvKysApFOXbLxBv81z6wg+jRcQmOqzXgzKHZwOVC3x4c4CIMBn2sRoDF70QzZwZ7iOvHiRh53wQvSlbELL4aWl1Qy5i0nHkS6ZGAXW1YN5J5LJKuG0X6mGM/FLBsq8PNZDY1VmpK1ZKTT7b22XP/+NJVb5vgD4mbctzurpmm/VYYF40izF4Zmr7sAgL57LM+CYkeh0IIM/ghl2ih57PTYNbAZIgVy+7XwVRloEAbbyos+yNU9Fq3zOH+syYQz4AimJelhdre8DOGVDPnXsSZUrGCBXgjYJF1W52lmhZ69XaAfpJj1l9bZPDl586+Jtr9W3J1fTvgCA/d4fXdysdwTfRQcC/HJsAAatAKyZ2gMDnhRXr1wRoIs/BUbI1mkfX4ceuS4CiVx6PXhMk5HeBnIhL2KXR2dM6b6RdEQ5W2bBSqujwjTJoqWQKYnQrdpKvtqtYJmWPSx+5aVMvRqt8lgvOfLiV3nRvmvxY//PzVTnaW0AIPSzP7b5ls3l4ocBwYczHC/D/gzfezpDDmDiTKB92zIh3/dw7+W8/4ceMz3OBKHrB0PY12vfT9s+H9j1KZdtESRTANGn6Ad98cXgM03bij0iqmSkn64usZ5PqvsliheW4t0FlkwPYt4bCHP0xkm8ukFU9fX7PtxJJ2RVB2Q3IorbSRkLQJvJdLoKWnk0+39I4L+lyPP8wABA8PI/sflX5ex3lWNxtt8Sitcdrl4FIKIZjODVjMUOZXpP1x0kysNmPWSBDPrQk5fyYzsFtkScLJs6QaGd0E9v2V7w8hWlWYreQazWU3smG9ZDjlCeJiC93J96EMCaCcyqA//ARqMBFCso5kZmdYH/rvZ/fuXrQnH966EBwB3CS75h55sEzA8FyDGkACUc7Vmthgv8kqsRxde60dM1BMg4zLCHjN0asvZYtRfQiNF51Yca1lQPL1B3om8lVIRkrZAPrEy9i97EyCYTo3WVKyr2tTrK0dgoqyoG+xlhJluAo3ZUyo5ppfuh9g1f8U11x+8gtVlLB4m19ie/+czrBOAPaAXgfOhZzfJtgAFWV4Ef/OBBi5tCsUW4LhpbRH3ix+NVsS3E27+4TxBl5HlOgMs6pVv6ard4PAXEChN1Pc2Drus86Bk2eDrXvyxWdfcb2eS7vZRnOyp70os/IkHfWbHCPqsWP7ZY46pw4Vu3UQ5UY+tRuTxOLuUeqOV6L4VUZLhkqdLQSj1p8Ehlz39vZvldi39z8LIfCvF66AowCv70vzn5Fj1i/TVy5K0FEM71rLYTy/EAVU4KYO0IyxQvgLHT0gkVVNZNe6MdbFgex+gqm86zTn/tB9en8kSndHgCLlogFTxjHatRr1xymqElE62GDfSrHgKql6Bz6oNMNDqTgajkVQDZTOjPdBfLjVvbYvNrjgs+lo4dAAj/u3ecuvm5V136u4TZG3zHUJ2ImY+h0V0TQIymVofoPrw4DOK2CPhh6Rf4lcI1Ew9615nERONftFO6zrste2uFta4ymZyBEtalMgEwyU6WJtoBcgZM8iWoaPPbzKpjirLraWMtD0EleF5UdPd2sfGGdtV1v2vxjn9wM6zjpqmnx9VIuT/7px65UX8a+fXqxKsWbfdEBABg6Z2CiHGKT/C8VTD7A8BY+lVXJHmW++Sfy3yf/dThYyMe+KSMvJd4bFqPOnI8wDnpIMuPRNu+l+vkHbIFlP16uNTbkvrj9vJdCrwIwrJN4NVYoMUY66mj8AtBH0jWh1h2FL6Uw8xjia+EE+tCj3Ill4Om4lkd89/RTmy8efGv/v7KHb4SPyofTR8lu5b/Ld/y2NXLM+deqz+q+BqthVcxmAoAL+sDoOa5jsPCWXa6gQqA4lwRwAfY2OMPNha4CQg6CTCgEAB9v3YfJJ/82LupS1cfZ8Zj49gLWuzn2OCKx8ApW38lAKJtjwu+ZA0wY/J4AmjGCR3gzKesC2d3sAfPOyjGAMDT8L0+o6u0It/u13Ppb28nT7518WM39Q92QvDiXgezF6c4l37nO5ebP/euz79isbf79fr1zpfrLKLvG+5t44C4NHMECE6zg+Qwl5MWgVCy5PDjcGVgcwZiyzNSzyEEHz/F7O8860o/AyBsYU8Ay9EHB0C0j13LZrDyUzZ1nyJAj2CxjPpDcjAquOKmVvQfeh3+DDxyENEB8MH7a4MglhNs6Buky49IXo9xLfQkz+95/+Kn4vN8m3sKL0MXnoKVNao3fcvy5INn77pxubt4qb7te72+FHmNAuIajfvZOjVfoV94u0wBcbkcekqrh76fsNyUQ/UXTMgrAGK5DUfH7AwwJxDxkQPBgTTKVAAAfNrLWbo+AEImgmg1AHwTKwOwbNEPQKVvOLECIHANW3auXrDJqkHSuHc1xgsisKzp14eW/DD6o/qZwsckph8KWn5O+d2i363l6nbxPtEu37lVM52fnfqCp/8JEycDApvcoj0AAAAASUVORK5CYII=';
```

Comment:
This looks pretty large, any chance you can provide an embedded SVG?

Reply:
I can't use a SVG due to the gradient, but I have updated to a smaller png: 20kb -> 10kb